remove v1 vs v2 mentions (#14)
unicamp-dl/mMARCO: README_old.md

# mMARCO

> ** This README refers to the old version (v1) of mMARCO. We highly recommend you use the latest dataset and models from the main [README.d] **

Reviewer: ```suggestion
**This README refers to the old version (v1) of mMARCO. We highly recommend you use the latest dataset and models from the main [README.md](/README.md)**
```

> ## Re-ranking with mT5

Reviewer: ```suggestion
## Re-ranking with PTT5
```

> Finally, we can re-rank our BM25 initial run using [mT5-base-multi-msmarco](https://huggingface.co/unicamp-dl/mt5-base-multi-msmarco) (or each one of the previous listed models):

Reviewer: ```suggestion
Finally, we can re-rank our BM25 initial run using [ptt5-base-en-pt-msmarco-10k](https://huggingface.co/unicamp-dl/ptt5-base-en-pt-msmarco-10k) (or each one of the previous listed models):
```
